tensorflow/tools/compatibility/ipynb.py

```diff
@@ -75,15 +104,17 @@ def _get_code(input_file):
     if is_python(cell):
       cell_lines = cell["source"]
 
+      is_line_split = False
       for line_idx, code_line in enumerate(cell_lines):
 
         # Sometimes, jupyter has more than python code
         # Idea is to comment these lines, for upgrade time
-        if code_line.startswith("%") or code_line.startswith("!") \
-            or code_line.startswith("?"):
+        if skip_magic(code_line, ['%', '!', '?']) or is_line_split:
           # Found a special character, need to "encode"
           code_line = "###!!!" + code_line
```

Comment:
I don't understand how this works...

Reply:
The idea behind is just to make it work for cases like 

```
!gcloud ml-engine models create ${MODEL} \
        --regions us-central1
```

So I just check if the "magic" code line ends with \, so I also need to comment the next line, since it's not python 🙂

Reply:
I also prepared a couple of test case, that I am planning to add somewhere, so it's testable - > https://colab.research.google.com/drive/1fCcmg8ZcbnR5dnG3KNl96QKN7u0FrQL-#scrollTo=jrxeKRCwogn6

Reply:
Yes, would be good to have test cases for this stuff with the code here.

I see the logic to skip the next line, but what does the `###!!!` do?

Reply:
It's just a way to encode these sections with unique commenting prefix, so I can strip them after pasta. 

`###!!!` it's just "unique pattern" that is less likely to be used by somebody else. Otherwise I would remove something, that I did not want to remove